k8s/aat/common/xui/xui-tc.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: xui-terms-and-conditions
+  namespace: xui
+  annotations:
+    fluxcd.io/automated: "true"
+    filter.fluxcd.io/tag.node: glob:prod-*
+spec:
+  releaseName: xui-terms-and-conditions
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: terms-and-conditions
+    version: 1.0.0
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/xui/terms-and-conditions:prod-fb64c1ca
+      ingressHost: xui-terms-and-conditions-aat.service.core-compute-aat.internal
```

Comment:
It would be ideal to move this as templated to base chart, not a show stopper though.

Reply:
noted